draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -254,54 +170,55 @@ Alternative methods, such as post-handshake certificate-based client authenticat
+* Implementations SHOULD NOT keep a connection open for longer than the validity span of the endpoint certificate. At the time the endpoint certificate expires, the connection SHOULD be closed and re-opened.
 
-RadSec endpoints SHOULD NOT be pre-configured with a set of trusted CAs by the vendor or manufacturer that are enabled by default.
-Instead, the endpoints SHOULD start off with an empty CA set as the trust base.
+RadSec endpoints SHOULD NOT be pre-configured with a list of trusted CAs by the vendor or manufacturer that are enabled by default.
+Instead, the endpoints SHOULD start off with an empty CA list as trust anchor.
 The addition of a CA SHOULD be done only when manually configured by the administrator.
-This does not preclude vendors or manufacturers including their set of trusted CAs in their products, but the enabling of those lists should be a conscious decision by an administrator.
+This does not preclude vendors or manufacturers including their trust list in their products, but the enabling of those lists should be a conscious decision by an administrator.
 
-RadSec endpoints MUST follow {{!RFC9525}} when validating peer identities.
-Specific details are provided below:
+RADIUS/(D)TLS clients and servers MUST follow {{!RFC9525}} when validating endpoint identities. Specific details are provided below:
 
 * Certificates MAY use wildcards in the identifiers of DNS names and realm names, but only as the complete, left-most label.
-* RadSec clients validate the server's identity to match their local configuration, accepting the identity on the first match:
+* RadSec clients validate the servers identity to match their local configuration, accepting the identity on the first match:
   * If the expected RadSec server is associated with a specific NAI realm, e.g. by dynamic discovery {{!RFC7585}} or static configuration, that realm is matched against the presented identifiers of any subjectAltName entry of type otherName whose name form is NAIRealm as defined in {{!RFC7585, Section 2.2}}.
-  * If the expected RadSec server was configured as a hostname, or the hostname was yielded by a dynamic discovery procedure, that name is matched against the presented identifiers of any subjectAltName entry of type dNSName {{!RFC5280}}.  Since a dynamic discovery might by itself not be secured, implementations MAY require the use of DNSSEC {{!RFC4033}} to ensure the authenticity of the DNS result before considering this identity as valid.
+  * If the expected RadSec server was configured as a hostname, or the hostname was yielded by a dynamic discovery procedure, that name is matched against the presented identifiers of any subjectAltName entry of type dNSName {{!RFC5280}}. Since a dynamic discovery might by itself not be secured, implementations MAY require the use of DNSSEC {{!RFC4033}} to ensure the authenticity of the DNS result before considering this identity as valid.
```

Comment:
```suggestion
  * If the expected RadSec server was configured as a hostname, or the hostname was yielded by a dynamic discovery procedure, that name is matched against the presented identifiers of any subjectAltName entry of type dNSName {{!RFC5280}}.  Since a dynamic discovery might by itself not be secured, implementations MAY require the use of DNSSEC {{!RFC4033}} to ensure the authenticity of the DNS result before considering this identity as valid.
```